packages/calcite-components/src/components/list-item/list-item.tsx

```diff
@@ -27,7 +27,7 @@ import {
 } from "../../utils/interactive";
 import { SelectionMode } from "../interfaces";
 import { SelectionAppearance } from "../list/resources";
-import { CSS, ICONS, SLOTS } from "./resources";
+import { CSS, dataTestActiveCellAttr, ICONS, SLOTS } from "./resources";
```

Comment:
Very Nitpick: I think we try to avoid abbreviations, so should we do `dataTestActiveCellAttribute`? 
I get why it was abbreviated though, because that's pretty long lol

Reply:
lol yeah I agree about abbr. :)

 I'll rename.